src/payer.py

```diff
@@ -0,0 +1,76 @@
+from typing import List, NamedTuple, Optional, Union
+
+import solana.system_program as sys_lib
+from solana.account import Account
+from solana.publickey import PublicKey
+from solana.rpc.api import Client
+from solana.rpc.commitment import Commitment, Max, Single
+from solana.rpc.types import RPCResponse
+from solana.transaction import Transaction, TransactionInstruction
+
+
+class UninitializedAccount(NamedTuple):
+    account: PublicKey
+    """Public key of an uninitialized account."""
+    instruction: Union[TransactionInstruction, Transaction]
+    """Transaction or transaction instruction to execute to intialize the account."""
+
+
+class Payer(Account):
```

Comment:
Not sure if we need this. At the minimum, I am not sure if this should be part of the public API. The API user should provide a public key for fee-paying each time they send an instruction to the DEX is my understanding.